### PR TITLE
Add new command to count number of times a certain license is used

### PR DESCRIPTION
**Pull request by jdrieghe**
### Added
- Command to count the number of times a certain license is used in the codebase

---
image

### CODE REVIEW

File: src/Composer/UsedLicensesParser.php [hunk 1]
```diff
@@ -49,4 +49,26 @@ public function getPackagesWithLicense(string $license): array
...
+            if (isset($dependency['license'][0])) {
+                $licenseName = $dependency['license'][0];
+                if (!isset($licenses[$licenseName])) {
+                    $licenses[$licenseName] = 0;
+                }
+                $licenses[$licenseName]++;
+            }
+        }
+
+        asort($licenses, SORT_NUMERIC);
```
Comment (stevetauber) [inline src/Composer/UsedLicensesParser.php:70]: Why not arsort? 

https://www.php.net/manual/en/array.sorting.php

File: src/Commands/CountUsedLicenses.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription('Count number of dependencies for each license');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
```
Comment (stevetauber) [inline src/Commands/CountUsedLicenses.php:36]: Nice use of this :)

File: src/Commands/CountUsedLicenses.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            $output->writeln($e->getMessage());
+            return 1;
+        }
+
+        $io->table(
+            ['License', 'Number of dependencies'],
+            $rows
+        );
+
+
```
Comment (WouterSioen) [inline src/Commands/CountUsedLicenses.php:53]:
```suggestion
```

File: src/Composer/UsedLicensesParser.php (hunk 1 above)
Comment (jdrieghe) [reply src/Composer/UsedLicensesParser.php:70]: Because I never bothered learned all sort methods by heart 😂

**Comment by jdrieghe** [comment]
Going to merge it. The added test is not going to hurt anyone and I don't think it needs an explicit re-review.

**Pull request merged**